contracts/sdk/IpcContract.sol

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
@raulk should this file be renamed to `IpcExchange.sol` or `IpcHandler` to avoid introducing more names?

Reply:
I think we can separate the interface and the abstract contract into two separate files.